src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -270,10 +270,12 @@ const mapStateToProps = state => {
     isInMVI: isInMVISelector(state),
     isLOA3,
     shouldFetchDirectDepositInformation,
+
     shouldShowDirectDeposit:
-      shouldFetchDirectDepositInformation &&
-      !isDirectDepositBlocked &&
-      (isDirectDepositSetUp || isEligibleToSignUp),
+      (isLOA3 && !is2faEnabled) ||
```

Comment:
Technically we should add a `&& isEvssAvailable` check to this, BUT I actually like this better because it just means more folks will be prompted to set up 2FA which is a good thing.